cfpb/cfgov-refresh
# Removing jQuery

Removing jQuery and it's associated files.

## Additions

- Added Carousel code ( this code is just minor modifications to the SwipeJs code ).

## Removals

-  Removing jQuery and it's associated files.


## Changes

- Modified less file to be current with DOM modifications.

## Testing
- Run 'gulp build`
-  Visit `http://localhost:8000/about-us/the-bureau/bureau-structure/` and it should perform as expected.

## Review

- @cfpb/cfgov-frontends 

## Notes

- `SwipeJS` currently needs some modifications, so I've included that code internally. It might make sense just to fork it.
- We still load jQuery from the CDN because it's used by the analytics team and in other apps.

## Todos

-

## Checklist
